### ISSUE TITLE
Entering the Aether on a Server, Crashes the client

### ISSUE THREAD

**Issue by themodernmagister** [Modding-Legacy/Aether-Legacy#331]
Aether Legacy v3.2
Forge 14.23.4.2739
Mod list: http://modlister.com/user/Modern_Magician

Issue: Client Crashes when entering server presumably, because I entered the Aether with incompatible mods. Something to do with the Swet mob.

Crash Log: https://paste.ee/p/TPpIG

**Comment by Lachney** [comment]
An issue with swets, the best thing you can temporarily do for now is to try and remove swets from the world before you join inside the world:

For future reference

````java.lang.ClassCastException: java.lang.Integer cannot be cast to java.lang.Boolean
	at com.legacy.aether.entities.passive.mountable.EntitySwet.isFriendly(EntitySwet.java:622)
	at com.legacy.aether.entities.passive.mountable.EntitySwet.updateRider(EntitySwet.java:167)
	at com.legacy.aether.entities.util.EntityMountable.func_70071_h_(EntityMountable.java:69)
	at com.legacy.aether.entities.passive.mountable.EntitySwet.func_70071_h_(EntitySwet.java:105)``

**Comment by LemADEC** [comment]
this is a dup of #311

**Comment by Cakx** [comment]
I had the same issue today using 3.2. (Swet error / ticking)
Just 30 seconds after I entered the Aether for the first time the client crashed and I wasn't able to join my server again (client crashed again on world load - every time).  

I then deleted the dimension files (folder: Dim-Aether) from my server which 'solved' the problem by recreating the dimension on my next join attempt. _(Which of course also deletes the 'Back'Portal!)_

Worked fine from then on but I feel like this could happen any time again. 
Is there any other way besides deleting/resetting the whole dimension? I wasn't able to figure out how i could prevent Swets from spawning / remove them from my world.
Hope it gets fixeds soon.
